frontend/src/pages/home/components/UpcomingMeeting.tsx

```diff
@@ -61,6 +76,48 @@ export default function UpcomingMeeting({ meeting }: UpcommingMeetingProps) {
         setIsLoading(false);
     }
 
+    function handleSubmit(formValues: TFormValues) {
+        if (formValues.startTime.getTime() < Date.now()) {
+            alert('Start time cannot be in the past');
+            return;
+        }
+
+        if (formValues.duration <= 0) {
+            alert('Duration must be positive');
+            return;
+        }
+
+        editMeeting(formValues);
+    }
```

Comment:
You reckon we can extract this to create a helper function as it overlaps with the one with creating meeting?

Reply:
I've removed the logic checking into modal as it didn't seem reusable enough for a util.
separate functions will handle creation of meeting tho